### PR TITLE
[Task2/3] WebElement methods and xpath

### PR DESCRIPTION
**Pull request by Zhukova-Iryna**

### CODE REVIEW

File: src/main/java/ua/price/pages/SearchPage.java [hunk 1]
```diff
@@ -18,6 +18,9 @@
     @FindBy(xpath = ".//div[contains(@class, 'product-block type')]")
     private List<WebElement> searchResultList;
 
+    @FindBy(xpath = ".//a[contains(@class,'btn-orange')]")
+    WebElement itemPageButton;
```
Comment (DmitriyBondarenko) [inline src/main/java/ua/price/pages/SearchPage.java]: Make it `private` please

File: src/main/java/ua/price/pages/UserAccountPage.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    public void logOut() {
+        clickWebElement(accountIcon);
+        clickWebElement(logoutLink);
+    }
+
+    public boolean isSuccessMessageAppears() {
+        return successMessage.isDisplayed();
+    }
+
+    public UserAccountPage(WebDriver driver) {
```
Comment (DmitriyBondarenko) [inline src/main/java/ua/price/pages/UserAccountPage.java]: Move this constructor under WebElements

File: src/main/java/ua/price/pages/SearchPage.java (hunk 1 above)
Comment (Zhukova-Iryna) [reply src/main/java/ua/price/pages/SearchPage.java]: Done

File: src/main/java/ua/price/pages/UserAccountPage.java (hunk 2 above)
Comment (Zhukova-Iryna) [reply src/main/java/ua/price/pages/UserAccountPage.java]: Done

**Review by DmitriyBondarenko — APPROVED** [review]

**Pull request merged**